src/swap/useSwapQuote.ts

```diff
@@ -126,6 +140,12 @@ function useSwapQuote(networkId: NetworkId, slippagePercentage: string) {
       swapAmount: ParsedSwapAmount,
       updatedField: Field
     ) => {
+      if (!walletAddress || !isAddress(walletAddress)) {
```

Comment:
nit: we usually aren't that defensive about walletAddress.
```suggestion
      if (!walletAddress) {
```

Reply:
same reason as above...walletAddress has type string which means later I need to type cast it to `Address` to use with viem...perhaps i should just change the type of walletAddress to be `Address` in redux, but that'll be a bigger change...